Assets/Scripts/SettingsManager.cs

```diff
@@ -443,7 +443,7 @@ public void LoadSettings()
             EnableSpellShadows = GetBool(sectionSpells, "EnableSpellShadows");
 
             InvertMouseVertical = GetBool(sectionControls, "InvertMouseVertical");
-            MouseLookSmoothing = GetBool(sectionControls, "MouseLookSmoothing");
+            MouseLookSmoothing = GetFloat(sectionControls, "MouseLookSmoothing");
```

Comment:
Don't forget to enforce valid range when reading a setting like this.

Reply:
Got it, thanks.